ubiquity/dell-bootstrap.py

```diff
@@ -615,6 +627,8 @@ def fixup_factory_devices(self, rec_part):
 
         self.device = rec_part["slave"]
 
+        # device = self.device
```

Comment:
Don't add a commented out line in a commit

Reply:
Got it